proto/errorpb.proto

```diff
@@ -148,4 +153,6 @@ message Error {
     DataIsNotReady data_is_not_ready = 13;
     RegionNotInitialized region_not_initialized = 14;
     DiskFull disk_full = 15;
+    // Online recovery is still in performing, reject writes to avoid potential issues
+    RecoveryInProgress RecoveryInProgress = 16;
```

Comment:
Why not use snake_case? For the sake of consistency 😅

Reply:
oh, miswrite